scripts/services/setConfig.ts

```diff
@@ -0,0 +1,14 @@
+import { matches, compat, types as T } from "../deps.ts";
+const {shape, literal} = matches
+const matchIsRest = shape({advanced:shape({plugin: shape({rest: literal(true)})})}).test
+
+// deno-lint-ignore require-await
+export const setConfig: T.ExpectedExports.setConfig = async (
+  effects: T.Effects,
+  newConfig: T.Config,
+) => {
+  const dependsOnCln: { [key: string]: string[] } = matchIsRest(newConfig) ? { "c-lightning": [] } : {};
```

Comment:
What does this do?

Also, where is LND?

https://github.com/Start9Labs/ride-the-lightning-wrapper/blob/3028153b611c0f80580fceac36340f22eb8c8583/assets/compat/dependencies.yaml#L1-L6

Reply:
I have no idea, this is not part of the PR

Reply:
looks like this is your code @chrisguida

Reply:
Yeah I was uncertain about this as well, and could not remember if RTL could have dependencies on both. Thanks for linking the prior version @chrisguida - seems like this piece is indeed missing

Reply:
ok @Blu-J helped me to add the missing dependencies.